lib/signaling/v2/peerconnection.js

```diff
@@ -263,7 +263,10 @@ PeerConnectionV2.prototype._answer = function _answer(offer) {
   }).then(function() {
     return self._peerConnection.createAnswer();
   }).then(function(answer) {
-    return self._setLocalDescription(answer);
+    return self._setLocalDescription({
+      type: answer.type,
+      sdp: answer.sdp
+    });
```

Comment:
This doesn't need to change, does it?

Reply:
@markandrus Yes it does, because `.createAnswer()` returns an `RTCSessionDescription`, whose members are read-only in Safari.

Reply:
Is `_setLocalDescription` mutating its input argument? If so, that might explain why I saw the read-only warnings in Firefox again recently after we had fixed them. Instead of letting `_setLocalDescription` mutate its argument, I think it should copy it, if necessary.